app/src/util/form-schema/index.ts

```diff
@@ -25,7 +25,7 @@ export function findMethodology(
 export interface ExtraField {
   id: string;
   type?: string;
-  options?: string[];
+  options?: string[] | { type: string; names: string[] }[];
```

Comment:
Ideally we only have one of these and rework the existing data to also use this schema. Otherwise we need to have an if/ else block for both cases in every place this is used...

Reply:
We could potentially have a helper function that normalizes it, but at that point let's just use my automatic rewriting script for the JSON and fix all of the data types to be consistent.